pkgs/development/python-modules/buildstream-plugins/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cython,
+}:
+
+buildPythonPackage rec {
+  pname = "buildstream-plugins";
+  version = "2.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "apache";
+    repo = "buildstream-plugins";
+    tag = version;
+    hash = "sha256-VAHDMy4DvNneWP1jhrIZzogZ5Gz5PS/GcFpOg2cGYvs=";
+  };
+
+  build-system = [
+    cython
+    setuptools
+  ];
+
+  # Do not run pyTest, causes infinite recursion as `buildstream-plugins`
+  # depends on `Buildstream`, and vice-versa for tests.
+  # May be fixable by skipping certain tests? TODO.
+
+  pythonImportsCheck = [ "buildstream_plugins" ];
```

Comment:
Why no tests ?
Please mention in a comment the reason, or enable them.

Reply:
I did explain in the comment above in that diff, but essentially, `buildstream_plugins` depends on `buildstream`, and vice versa. It would be a lot of tests to disable, rendering the result the same as disabling tests. I can't yet find a way around this.

Reply:
Hi @drupol, I've edited the comment in that diff, and I hope it's a bit more clear. The Buildstream package generally requires this in its `PYTHONPATH`.